src/index.js

```diff
@@ -1,46 +1,16 @@
 'use strict';
 
 var _ = require('underscore'),
-    DEFAULT_RESOURCE_CONFIG;
+    resourceUtils = require('./util/resource'),
+    constants = require('./constants');
 
-DEFAULT_RESOURCE_CONFIG = {
-   AbsoluteMinimumProvisioned: 1,
-   AbsoluteMaximumProvisioned: 10,
-   MinutesOfStatsToRetrieve: 20,
-   MinutesOfStatsToIgnore: 2,
-   MinutesToForecast: 5,
-   PreferredForecastPadding: [
-      { IfGreaterThan: 0, IfLessThanOrEqual: 30, Percentage: 100 },
-      { IfGreaterThan: 30, IfLessThanOrEqual: 100, Percentage: 50 },
-      { IfGreaterThan: 100, IfLessThanOrEqual: 1000, Percentage: 25 },
-      { IfGreaterThan: 1000, IfLessThanOrEqual: 1000000, Percentage: 15 },
-   ],
-   MinimumMinutesBetweenIncreases: 0,
-   MaximumIncreaseAmount: [
-      { IfGreaterThan: 0, IfLessThanOrEqual: 10, Percentage: 100 },
-      { IfGreaterThan: 10, IfLessThanOrEqual: 100, Percentage: 50 },
-      { IfGreaterThan: 100, IfLessThanOrEqual: 1000, Percentage: 20 },
-   ],
-   MinimumMinutesBetweenDecreases: 45,
-   MinimumDecreaseAmount: [
-      { IfGreaterThan: 0, IfLessThanOrEqual: 100, Percentage: 50 },
-      { IfGreaterThan: 100, IfLessThanOrEqual: 1000000, Percentage: 30 },
-   ],
-   MinimumMinutesBeforeDecreaseAfterIncrease: 15,
-   MaximumDecreasesToUsePerDay: 4,
-};
+module.exports = {};
 
+// Expose for easy access and legacy compatibility
+_.extend(module.exports, _.pick(constants, 'READ', 'WRITE', 'DEFAULT_RESOURCE_CONFIG'));
```

Comment:
Since we'll need to do a version bump anyway, let's just expose all constants as "constants" so someone can do `var DCM = require('dynamodb-capacity-manager'), constants = DCM.constants;`